pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -545,6 +545,59 @@
     patches = [ ./patches/coq_nvim/emulate-venv.patch ];
   };
 
+  cord-nvim =
+    let
+      version = "2024-07-19";
+      src = fetchFromGitHub {
+      owner = "vyfor";
```

Comment:
I think that the indentation of the entire snippet is off.

Reply:
Should be good now, I formatted the snippet.